Firmware/RTK_Surveyor/Begin.ino

```diff
@@ -430,18 +452,21 @@ void endSD(bool alreadyHaveSemaphore, bool releaseSemaphore)
     if (USE_SPI_MICROSD)
       sd->end();
 #ifdef COMPILE_SD_MMC
-    else
-      SD_MMC.end();
+//    else
+//      SD_MMC.end(); // NO!! DO NOT!!
```

Comment:
Comment could be improved. "MMC does not support end()" or what have you.

Reply:
Indeed. Improvement required. (This was causing some bizzare file-corruption weirdness and I was very happy when I found it...!)